run_cv.py

```diff
@@ -47,10 +49,18 @@
     target_detector = LoadingBayDetector(generator)
 elif args.target == "power_port":
     target_detector = PowerPortDetector(generator)
+    pc = PoseCalculator(target_detector)
+elif args.target == "power_cell":
+    target_detector = PowerCellDetector(generator)
+    pc = BallFinder(target_detector)
 
-with PoseCalculator(target_detector) as pc:
-    while (generator.is_capturing() if args.generator == "video_file" else True):
+while generator.is_capturing() if args.generator == "video_file" else True:
+    if type(target_detector) is PowerCellDetector:
+        _, obstacle_present = pc.get_balls()
+        print(obstacle_present)
+    else:
         pc.get_values(units=args.units)
-        key = cv2.waitKey(wait_time)
-        if key == ord('q'):
-            break
+
+    key = cv2.waitKey(wait_time)
+    if key == ord('q'):
```

Comment:
Since I keep forgetting, can we print a message like "Press q to quit" before we start the run loop?